### PR TITLE
chore: import kintone/dts-gen

### PR DESCRIPTION
**Pull request by koba04**
# DO NOT SQUASH MERGE, USE MERGE COMMIT

## Why

We'd like to transfer `kintone/dts-gen` into `kintone/js-sdk`.

## What

- [x] package.json
- [x] README in root
- [x] README for dts-gen
- [x] build settings

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/js-sdk/blob/master/CONTRIBUTING.md)
- [ ] Updated documentation if it is required.
- [ ] Added tests if it is required.
- [ ] Passed `yarn lint` and `yarn test` on the root directory.

### CODE REVIEW

File: packages/dts-gen/docs/how-to-use.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+## How to generate kintone-dts-gen
+
+you can generate `sample-field.d.ts` like below:
+
+```bash
+$ kintone-dts-gen --host http://***.cybozu.com \
+                 -u username \
+                 -p password \
+                 --app-id 12 \
+                 --type-name SampleFields
```
Comment (chick-p) [inline packages/dts-gen/docs/how-to-use.md:26]: typo from before.
```suggestion
                 --type-name SampleFields \
```

**Comment by chick-p** [comment]
Thanks you for your PR!
I confirmed that this PR can generate a record field definition.

File: packages/dts-gen/docs/how-to-use.md (hunk 1 above)
Comment (koba04) [reply packages/dts-gen/docs/how-to-use.md]: Good catch, thanks!

**Review by shisama — APPROVED** [review]

**Review by chick-p — APPROVED** [review]
👍

**Pull request merged**